src/AspNetCoreModuleV2/InProcessRequestHandler/dllmain.cxx

```diff
@@ -41,6 +42,12 @@ InitializeGlobalConfiguration(
         }
 
         g_pHttpServer = pServer;
+        if (FAILED(hr = ALLOC_CACHE_HANDLER::StaticInitialize()) ||
```

Comment:
FINISHED_IF_FAILED

Reply:
I am not a big fun to use macro/script as it makes the code hard for read and debug (not just for us but also for CSS).  Updating it.

Reply:
But at least we would know what failed and where. Which makes debugging much easier